dashboard/app/helpers/levels_helper.rb

```diff
@@ -384,4 +384,14 @@ def enable_scrolling?
   def enable_examples?
     current_user && current_user.admin? && @level.is_a?(Blockly)
   end
+
+  # If this is a restricted level (i.e. applab) and user is under 13, redirect with a flash alert
+  def redirect_under_13(level)
```

Comment:
maybe say something about applab in the method name so people looking at this code don't expect <13 users to be redirected for all levels? redirect_under_13_away_from_applab may be a bit long tho